packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -572,6 +576,31 @@ export class FluidDataStoreRuntime
 		return this.dataStoreContext.uploadBlob(blob, signal);
 	}
 
+	/** @returns true if it found/processed GC Data, false otherwise */
+	private extractAndProcessAttachGCData(id: string, entries: ITreeEntry[]): boolean {
```

Comment:
This can be a helper function outside of the class and can be re-used in both the call sites right?

Reply:
That way they won't get out of sync.

Reply:
Yeah I was waiting for someone to say that. I can do it. The signatures of the GC function differ but I can deal with it